h2o-core/src/main/java/hex/grid/GridSearch.java

```diff
@@ -714,10 +737,6 @@ public MP build() {
     }
   }
 
-  /**
-   * Constant for adaptive parallelism level - number of models built in parallel is decided by H2O.
-   */
-  public static final int ADAPTIVE_PARALLELISM_LEVEL = 0;
```

Comment:
have not found usages for this constant

Reply:
should be used in getParallelismLevel below